### ISSUE TITLE
Multiple calls to expectExceptionMessage

### ISSUE THREAD

**Issue by Seldaek** [sebastianbergmann/phpunit#6020]
I have a long exception message which has a lot of garbage in, but a few elements I want to check the presence of.

So I figured I'd call `self::expectExceptionMessage()` several times to assert these elements, however it seems that only the last call takes effect. That makes sense as that method just sets the expectedExceptionMessage property, but I thought it might be nice if it would collect a list of assertions to make instead of just overwrite.

Anyway as a workaround to make two assertions I now have expectExceptionMessage + expectExceptionMessageMatches, as both are preserved and checked.

**Comment by staabm** [comment]
with the current means you could do something like
```
try {
  $foo->doFoo();
} catch (MyException $e) {
  $this->assertContains('string1', $e->getMessage());
  $this->assertContains('another string', $e->getMessage());
  $this->assertContains('a 3rd string', $e->getMessage());
}
```

**Comment by Seldaek** [comment]
Right that'd have been my next option if I had to but I figured it could be nice if PHPUnit accepted multiple, or if not maybe it should throw if something is set already because it's kinda swallowing assertions silently right now.

**Comment by sebastianbergmann** [comment]
I do not think we should add more expectation variants for exception messages.
